pkg/kubelet/pluginmanager/operationexecutor/operation_generator.go

```diff
@@ -122,6 +125,9 @@ func (og *operationGenerator) GenerateRegisterPluginFunc(
 		if err := handler.RegisterPlugin(infoResp.Name, infoResp.Endpoint, infoResp.SupportedVersions); err != nil {
 			return og.notifyPlugin(client, false, fmt.Sprintf("RegisterPlugin error -- plugin registration failed with err: %v", err))
 		}
+		if len(pathToHandlers) != 0 {
```

Comment:
Maybe I missed something, because I'm confused the purpose of condition:` len(pathToHandlers) != 0 `, and I can't see where it is filled, so I think  pathToHandlers map will be empty if the code goes here for the first time.

Reply:
Take a look at line 70 in reconciler.go

Reply:
Both pathToHandlers map empty<make(map[string]cache.HandlerWithName)> and nil are let the condition: `len(pathToHandlers) != 0` to `true`.

Reply:
I read your comment more than once.
I don't think so - what you described was for len(pathToHandlers) == 0.

Reply:
sorry, typo, I have updated it. And I mean that the condition: `len(pathToHandlers) != 0` will let the first register plugin info can't add into pathToHandlers map because empty map (not nil) will also let  len(pathToHandlers) as 0